src/ResourceParser.h

```diff
@@ -133,6 +133,13 @@ namespace snowcrash {
                             attributes.sourceMap.name.sourceMap = out.sourceMap.name.sourceMap;
                         }
                     }
+                    else {
+
+                        mdp::CharactersRangeSet sourceMap = mdp::BytesRangeSetToCharactersRangeSet(node->sourceMap, pd.sourceData);
+                        out.report.error = Error("attributes can not be added to a resource which doesn't have a name",
```

Comment:
Also why is this the case? I believe this is incorrect. Following should be valid syntax (albeit it can't be referenced later):

```
# /resource
+ attributes
    + name (string)
```